### PR TITLE
Ravi/backend integration help

### PR DESCRIPTION
**Pull request by ravioli0509**
This PR integrates backend with the books in owners' tabs. 
Each book can be edited/deleted by after selecting the option from popup menu after clicking the 3 dot button on each book card.  
Also, implemented a logic in the book controller that would get all the books that the user is borrowing from other owners.

This PR also implements validation checks in editing book. Issue #75

### CODE REVIEW

File: app/src/main/java/com/team41/boromi/book/OwnedFragment.java [hunk 1]
```diff
@@ -35,6 +38,7 @@
   private PagerAdapter pagerAdapter;
   private BookActivity bookActivity;
   private String parent = "Owned";
+  private String TAG = "OwenedFrag";
```
Comment (blchelle) [inline app/src/main/java/com/team41/boromi/book/OwnedFragment.java]: OwnedFrag. Not really a big deal though...

**Review by blchelle — COMMENTED** [review]
Lol I'd say lgtm but honestly that went over my head. Someone else should review this as well

**Comment by ravioli0509** [comment]
@akchen1 I found another way around for that require api level thing for showing icon in the menu item. 
Should be good now :)

File: app/src/main/java/com/team41/boromi/book/OwnedFragment.java (hunk 1 above)
Comment (ravioli0509) [reply app/src/main/java/com/team41/boromi/book/OwnedFragment.java]: good catch! resolved

**Review by akchen1 — APPROVED** [review]
lgtm

**Pull request merged**